externalpackages/bin/git-archive-all.sh

```diff
@@ -0,0 +1,315 @@
+#!/bin/bash -
```

Comment:
One thing: This file doesn't say who it's by. Probably should credit https://github.com/meitar (interesting character btw https://maymay.net)

Reply:
Ah maybe I should just add it as a submodule?

Reply:
Is it a hard dependency? If not then perhaps a submodule might be better (?) No strong feelings about it. Otherwise maybe just adding a line to the file with a link to the original repository.